src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -435,7 +433,7 @@
 
 (react/defc Page
   {:render
-   (fn [{:keys [props]}]
+   (fn []
      [:div {:style {:marginTop "1.5rem"}}
       [WorkspaceList]])})
```

Comment:
Not easily. All Page elements are called as react components when they're rendered. We'd have to build an exception.